generators/_init-hybrid/templates/_gitignore

```diff
@@ -4,7 +4,6 @@
 /build/
 /dist/
 *.egg-info/
-*.egg
```

Comment:
@anita-steiner Why do we want to remove the `*.egg` and thus commit these files?

Reply:
because we use wheel instead

Reply:
That is true. We can remove this line, since we do not create `.egg` files during our build anymore.